user/dolphin-emu/patches/system-libs.patch

```diff
@@ -1,12 +1,12 @@
 diff -ruN a/CMakeLists.txt b/CMakeLists.txt
---- a/CMakeLists.txt	2024-12-02 00:00:52.000000000 +0100
-+++ b/CMakeLists.txt	2024-12-02 10:50:46.912016246 +0100
-@@ -672,10 +672,10 @@
+--- a/CMakeLists.txt	2025-06-05 08:52:15.000000000 +0200
++++ b/CMakeLists.txt	2025-06-13 17:47:47.334890260 +0200
+@@ -689,10 +689,10 @@
  
  dolphin_find_optional_system_library_pkgconfig(ZSTD libzstd>=1.4.0 zstd::zstd Externals/zstd)
  
--add_subdirectory(Externals/zlib-ng)
-+dolphin_find_optional_system_library_pkgconfig(ZLIB zlib-ng ZLIB::ZLIB Externals/zlib-ng)
+-dolphin_find_optional_system_library_pkgconfig(ZLIB zlib>=1.3.1 ZLIB::ZLIB Externals/zlib-ng)
```

Comment:
hm, is this still necessary? `zlib-ng-compat-devel` provides `pc:zlib` which i think should satisfy the original dep